astroid/brain/brain_functools.py

```diff
@@ -101,7 +101,9 @@ def _functools_partial_inference(node, context=None):
     partial_function = objects.PartialFunction(
         call,
         name=inferred_wrapped_function.name,
-        doc=inferred_wrapped_function.doc,
+        doc=inferred_wrapped_function.doc_node.value
```

Comment:
It seems like both `PartialFunction` and `Property` should accept `doc_node` instead.

Reply:
Not in their `init` right? They share the `postinit` from `FunctionDef` and don't redefine it, so I don't think we actually need to change those classes? We just need to pass different arguments to their `postinit`.